packages/node-core/src/indexer/store.service.ts

```diff
@@ -303,7 +306,8 @@ export class StoreService {
           });
           extraQueries.push(
             commentConstraintQuery(`"${schema}"."${rel.target.tableName}"`, fkConstraint, tags),
-            createUniqueIndexQuery(schema, relatedModel.tableName, relation.foreignKey)
+            createUniqueIndexQuery(schema, relatedModel.tableName, relation.foreignKey),
+            constraintDeferrableQuery(model.getTableName().toString(), fkConstraint)
```

Comment:
I think this should work without `constraintDeferrableQuery`.

It should pick up the same fkConstraint from `belongTo`, for both `hasOne` and `hasMany`

![image](https://user-images.githubusercontent.com/10431657/229648103-9fc66534-da0b-4af8-b329-fdc770468b6c.png)

![image](https://user-images.githubusercontent.com/10431657/229648124-0f8a2e10-59a5-4a21-aa1d-8798c071ae25.png)